### PR TITLE
Fix keyboard shortcuts order for MainTabBarController

### LINKED ISSUE #1951: Keyboard shortcut for Up Next is missing
### Description

The tab bar has the following tabs:

- Podcasts
- Filters
- Discover
- Up Next
- Profile

When using an iPad with a keyboard, the keyboard shortcuts relate to the tab, so command+1 is Podcasts, command+2 is Filters etc.

The issue is that Up Next is skipped, it should be command+4 but that goes to Profile.  Also holding the command button to see the shortcuts on screen shows that command 4 is for Profile and not Up Next

### Step-by-step reproduction instructions

1. Open pocket casts with an iPad keyboard attached
2. Hold the command button, see that in the pop up that there is no shortcut for Up Next
3. Press command+1, it correctly navigates to Podcasts
4. Press command+4 and see that it navigates to Profile and not to Up Next

### Screenshots or screen recording

_No response_

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

iPad Air 3, iOS 17.5.1, Pocket Casts 7.68.0.1

### PR DESCRIPTION
**Pull request by rviljoen**
Fixes #1951 

Keyboard shortcuts were not updated with the addition of Up Next to tab bar (see #1613). This PR adds a keyboard shortcut for the Up Next tab, and corrects the shortcut for the Profile tab.

## To test

1. Run on iPad
2. Ensure that `upNextOnTabBar` feature flag is enabled
3. Hold Cmd on keyboard (works fine in sim). Ensure that listed keyboard shortcuts for Tab Bar (Cmd+1, Cmd+2, etc) lines up with the actual order on the tab bar
4. Test each shortcut (Cmd+1 through Cmd+5) and ensure that navigation goes to the correct tab.

image

5. Disable `upNextOnTabBar` feature flag.
6. Kill and restart the app
7. Up Next should no longer be on tab bar. Hold Cmd and ensure that shortcut for Up Next is also gone.
8. Test each shortcut (Cmd+1 through Cmd+4) and ensure that navigation goes to the correct tab.

## Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [x] I have considered adding unit tests for my changes.
- [ ] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

File: podcasts/MainTabBarController+shortcuts.swift [hunk 1]
```diff
@@ -33,7 +33,10 @@ extension MainTabBarController {
         let discoverCommand = UIKeyCommand(title: L10n.discover, action: #selector(handleDiscover), input: "3", modifierFlags: [.command])
         addKeyCommand(discoverCommand)
 
-        let profileCommand = UIKeyCommand(title: L10n.profile, action: #selector(handleProfile), input: "4", modifierFlags: [.command])
+        let upNextCommand = UIKeyCommand(title: L10n.upNext, action: #selector(handleUpNext), input: "4", modifierFlags: [.command])
```
Comment (rviljoen) [inline podcasts/MainTabBarController+shortcuts.swift]: Do we need to make this change conditional based on the FeatureFlag, similar to here:
https://github.com/Automattic/pocket-casts-ios/blob/e2160c7f681e300105fe98f17043aa689a9bd114/podcasts/MainTabBarController.swift#L52-L58

Comment (SergioEstevao) [reply podcasts/MainTabBarController+shortcuts.swift]: Yes, please do.

**Comment by danielebogo** [comment]
Thanks @rviljoen ! It's on my todo list for today!

**Comment by rviljoen** [comment]
> Thanks @rviljoen ! It's on my todo list for today!

@danielebogo No rush! I think this is a low priority one.

**Review by danielebogo — APPROVED** [review]
All good during my test! :shipit: 
Thanks for doing this!

**Pull request merged**